test/authenticated/editing-notepad.js

```diff
@@ -0,0 +1,98 @@
+const assert = require( 'assert' );
+const { toggleFeature } = require( '../test' );
+const clipboardy = require( 'clipboardy' );
+
+const SAMPLE_RELEASE = 'https://www.discogs.com/release/edit/7796190';
+
+async function notepadIsLoaded( page ) {
+  await Promise.all( [
+    page.goto( SAMPLE_RELEASE, { waitUntil: 'networkidle2' } ),
+    page.waitFor( '#de-editing-notepad' )
+  ] );
+}
```

Comment:
Thanks for the thorough testing 😄

Reply:
I actually held off, I was gonna write more 😶